pyart/graph/radarmapdisplay_cartopy.py

```diff
@@ -108,7 +108,8 @@ def plot_ppi_map(
             lat_lines=None, lon_lines=None, projection=None,
             min_lon=None, max_lon=None, min_lat=None, max_lat=None,
             width=None, height=None, lon_0=None, lat_0=None,
-            resolution='110m', shapefile=None, edges=True, gatefilter=None,
+            resolution='110m', shapefile=None, shapefile_kwargs={},
```

Comment:
shapefile_kwargs should be None and an empty dictionary created in the function.  [Mutable default arguments are dangerous](https://python-guide.readthedocs.io/en/latest/writing/gotchas/#mutable-default-arguments)